src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Listeners/ScanBlobScanLogHybridPollingStrategy.cs

```diff
@@ -214,6 +214,7 @@ private void ThrowIfDisposed()
             // if starting the cycle, reset the sweep time
             if (continuationToken == null)
             {
+                containerScanInfo.CurrentScanBeginTime = DateTime.UtcNow;
```

Comment:
is the best name here "Scan" or "Sweep"? (both words are used interchangeably in this code)